src/ray/protobuf/common.proto

```diff
@@ -509,6 +509,10 @@ message TaskSpec {
   int64 generator_backpressure_num_objects = 38;
   // Boolean if task events enabled, i.e tasks events would be reported.
   bool enable_task_events = 39;
+  // If this task is originated from a detached actor,
+  // this field contains the detached actor id.
+  // Otherwise it's empty and is originated from a driver.
+  bytes root_detached_actor_id = 40;
```

Comment:
```suggestion
  bytes root_detached_actor_id = 40;
```
why don't we just call it `from_detached_actor: bool`?

Reply:
This way we can differentiate work processes belonging to different detached actors.